next-tavla/src/Board/scenarios/Table/components/Time/components/FormattedTime.tsx

```diff
@@ -4,7 +4,7 @@ import classes from '../styles.module.css'
 function FormattedTime({ time }: { time: string }) {
     return (
         <>
-            <div className="textRight weight600">
+            <div className="text-right font-semibold">
```

Comment:
vi kan kanskje lage variabler for font-weight (i fremtiden). Det virker litt lettere å bruke tall fremfor 'semibold' og 'thin' 💭

Reply:
Noterer det som en oppgave:D